checker/check_request.go

```diff
@@ -28,4 +28,6 @@ type CheckRequest struct {
 	OssFuzzRepo clients.RepoClient
 	Dlogger     DetailLogger
 	Repo        clients.Repo
+	// UPGRADEv6: return raw results instead of scores.
```

Comment:
There are a lots of `// UPGRADEvX:` comments spread across the code. Can we instead create an issue and update the comment here to `// TODO(#issue): ...`?

Reply:
so there are just 3 `// UPGRADEv6`: I'm keeping them jut because there's no issue or things to fix, it's just indicating as comment that it will be released for v6. I think the ones mentioning `v2` can be cleaned up as you suggested.